api/app/controllers/users/confirmations_controller.rb

```diff
@@ -2,15 +2,18 @@ module Users
   class ConfirmationsController < Devise::ConfirmationsController
     # GET /users/confirmation?confirmation_token=
     def show
-      self.resource = resource_class.confirm_by_token(params[:confirmation_token])
+      resource = resource_class.confirm_by_token(params[:confirmation_token])
       yield resource if block_given?
 
-      if resource.errors.empty?
-        redirect_to 'https://www.google.com', allow_other_host: true
-      else
-        # TODO: redirect to generic error in front
-        redirect_to 'https://www.error.com'
-      end
+      return unless resource.errors.empty?
```

Comment:
Si hay un error como se comporta ahora mismo? Ejecuta el `return` pero no hay redirect a ningún lado, no? Queda blank la página?
Capaz mismo no es necesario hacer una diferenciación si hay un error... y es más cosa de mandar el error como un parámetro para que lo agarren desde el FE.
O capaz sí diferenciamos:
- Si está ok -> redirect to login
- Si hay error -> redirect a la home page

Reply:
Yo haría la diferencia si. Como no esta la pag de error aún había dejado el return. Ahora cambie para que mande al inicio hasta que este la pagina de error generada